src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -424,10 +424,8 @@ def run_safety_checker(self, image, device, dtype):
 
     def decode_latents(self, latents):
         warnings.warn(
-            (
-                "The decode_latents method is deprecated and will be removed in a future version. Please"
-                " use VaeImageProcessor instead"
-            ),
+            "The decode_latents method is deprecated and will be removed in a future version. Please"
+            " use VaeImageProcessor instead",
```

Comment:
Could this be happening for a different version of `black` installed on the local machine? This should not have happened.

Reply:
This didn't happen with black but rather on `make fix-copies` 🤔